app/controllers/skills_controller.rb

```diff
@@ -0,0 +1,43 @@
+class SkillsController < ApplicationController
+
+  # GET /skills
+  def index
+    @skills = Skill.all
+    json_response(@skills)
+  end
+
+  # GET /skill/:id
+  def show
+    json_response(@skill)
+  end
```

Comment:
For `#show, #update, and #destroy`, `@skill` is currently undefined and gives an error in Postman